include/mbgl/style/layers/layer.hpp.ejs

```diff
@@ -44,6 +44,13 @@ public:
 <% } -%>
 
 <% } -%>
+    // Visibility
+    void setVisibility(VisibilityType) final;
```

Comment:
Let's discuss making that style change separately. We already have quite a few uses of `final` without `override`.